mypy/fastparse.py

```diff
@@ -120,7 +120,7 @@ def is_no_type_check_decorator(expr: ast3.expr) -> bool:
     return False
 
 
-class ASTConverter(ast3.NodeTransformer):
+class ASTConverter(ast3.NodeTransformer):  # type: ignore
```

Comment:
Can you open a typeshed issue to upgrade the typed_ast stubs, and link this (and similer ignores later) to it?